.github/labeler.yml

```diff
@@ -0,0 +1,27 @@
+version: 1
+labels:
+  - label: 'tag: new feature'
+    title: '^feat(\(.+\))?:.*'
+
+  - label: 'tag: bug fix'
+    title: '^fix(\(.+\))?:.*'
+
+  - label: 'tag: polish'
+    title: '^refactor(\(.+\))?:.*'
+
+  - label: 'tag: documentation'
+    title: '^docs(\(.+\))?:.*'
+  - label: 'tag: documentation'
+    files:
+      - ^website/docs/.*
+      - ^website/versioned_docs/.*
+      - ^website/src/data/users.js
+
+  - label: 'tag: internal'
+    title: '^(chore|misc)(\(.+\))?:.*'
+  - label: 'tag: internal'
+    files:
+      - '^.github/*'
```

Comment:
With the new "tag: maintenance" it's even more complicated. I think there's more human judgement here than any simple condition.

Reply:
So, instead of `internal` we have to use `maintenance` now?

Reply:
Let's keep it. "chore" and "misc" are more likely to be unlisted.